ciao-controller/internal/datastore/sqlite3db.go

```diff
@@ -1196,7 +1196,7 @@ func (ds *sqliteDB) getInstances() ([]*types.Instance, error) {
 	)
 	SELECT	instances.id,
 		instances.tenant_id,
-		IFNULL(latest.state, "pending") AS state,
+		IFNULL(latest.state, "` + payloads.ComputeStatusPending + `") AS state,
```

Comment:
There are a couple of other references. Should they also be replaced.

total_running AS
(
	SELECT	instances.node_id AS node_id,
		count(instances.instance_id) AS total
	FROM instances
	WHERE state='running'
	GROUP BY node_id
),
total_pending AS
(
	SELECT	instances.node_id AS node_id,
		count(instances.instance_id) AS total
	FROM instances
	WHERE state='` + payloads.ComputeStatusPending + `'
	GROUP BY node_id
),
total_exited AS
(
	SELECT	instances.node_id,
		count(instances.instance_id) AS total
	FROM instances
	WHERE state='exited'
	GROUP BY node_id
)